pkg/client/streams.go

```diff
@@ -123,7 +109,7 @@ func (c *immuClient) StreamGet(ctx context.Context, k *schema.KeyRequest) (*sche
 	value, err := stream.ReadValue(vr, c.Options.StreamChunkSize)
 	if err != nil {
 		if err == io.EOF {
-			return nil, stream.ErrMissingExpectedData
+			return nil, errors.New(stream.ErrMissingExpectedData).WithCode(errors.CodInternalError)
```

Comment:
if error declaration already includes the code it will be easier to define and read the codes associated to the errors. Also it will prevent the assignment of multiple codes to the same error

Reply:
_errors.New(message string)_ is the canonical way to trigger errors.
If we modify this signature it will not possible anymore to replace error standard package with the new "github.com/codenotary/immudb/pkg/errors"
I can add a map in which we store as a key the message and as a value the code.
This could help?

Reply:
fixed with the internal map as discussed